### PR TITLE
Feature/handle menu footer

### PR DESCRIPTION
**Pull request by Clr34**

### CODE REVIEW

File: src/components/BudgetCmpt.vue [hunk 1]
```diff
@@ -31,7 +31,35 @@
...
+                <tr>
+                  <td class="name"><div>{{ $t("TOTAL") }}</div></td>
+                  <td class="allocated">{{ getEurosAmount(this.totalBudgetData.allocated) }}</td>
+                  <td class="spent">{{ getEurosAmount(this.totalBudgetData.spent) }}</td>
+                  <td class="available">{{ getEurosAmount(this.totalBudgetData.available) }}</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
```
Comment (ermoca) [inline src/components/BudgetCmpt.vue:62]: not sure what happened here. Where was the whole budgetHeader before you add it here ?

File: src/views/AccountPage.vue [hunk 2]
```diff
@@ -171,6 +181,13 @@ export default defineComponent({
     },
     cancelEditing () {
       this.editingTitle = false
+    },
+    incomingOutgoingFlowClass (operation: Operation): string {
```
Comment (ermoca) [inline src/views/AccountPage.vue:185]: I wasn't sure of the role of the function by seeing the name in the template. 
I would suggest something like... getClassDependingOnAmount() ?

**Review by ermoca — COMMENTED** [review]
so much CSS ! You're gonna be an expert in responsive design by the end of this period !

File: src/components/BudgetCmpt.vue (hunk 1 above)
Comment (Clr34) [reply src/components/BudgetCmpt.vue:62]: placeholderTop must have a changing height (money to distribute or not), it the same part than div .fixed to be sure it's the same height and not having bugs.

File: src/views/AccountPage.vue (hunk 2 above)
Comment (Clr34) [reply src/views/AccountPage.vue:185]: changed

**Pull request merged**